openwisp_radius/tests/test_api/test_freeradius_api.py

```diff
@@ -1437,6 +1439,185 @@ def test_authorize_counters_exception_handling(self):
                 self.assertEqual(response.status_code, 200)
                 self.assertEqual(response.data, truncated_accept_response)
 
+    @mock.patch.object(app_settings, "ADDITIONAL_RADIUS_CHECKS", ["Simultaneous-Use"])
+    def test_authorize_simultaneous_use_within_limit(self):
+        """Test that user can authenticate when within simultaneous use limit"""
+        user = self._get_org_user().user
+        group = user.radiususergroup_set.first().group
+        self._create_radius_groupcheck(
+            group=group,
+            groupname=group.name,
+            attribute="Simultaneous-Use",
+            op=":=",
+            value="2",
+        )
+
+        # Create one open session (within limit of 2)
+        acct_data = self.acct_post_data
+        acct_data.update(
+            {
+                "username": user.username,
+                "unique_id": "test_session_1",
+                "stop_time": None,
+            }
+        )
+        self._create_radius_accounting(**acct_data)
+
+        # Authorization should succeed
+        response = self._authorize_user(auth_header=self.auth_header)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data["control:Auth-Type"], "Accept")
+
+    @mock.patch.object(app_settings, "ADDITIONAL_RADIUS_CHECKS", ["Simultaneous-Use"])
+    def test_authorize_simultaneous_use_exceeds_limit(self):
```

Comment:
what do you think of squashing this test with the previous test and have the assertions run in a `subTest`? That we we can reduce the amount of lines of code needed for this.